### ISSUE TITLE
The build in the CI is broken

### ISSUE THREAD

**Issue by mcagl** [DjangoGirls/tutorial#1855]
### Issue description

[This pipeline run](https://github.com/DjangoGirls/tutorial/actions/runs/11323400245/job/31485981412) did not build the site after the merge of [this PR](https://github.com/DjangoGirls/tutorial/pull/1852) that updated the tutorial to newer python/django, so the deployed website is still at the old versions.

### Language

It is a problem with the CI

### Operating system

It is a problem with the CI

**Comment by das-g** [comment]
The error seems to be
```
/home/runner/work/_temp/27dc039a-1a81-4838-beea-9079b31a6aea.sh: line 4: rename: command not found
```
at https://github.com/DjangoGirls/tutorial/actions/runs/11323400245/job/31485981412#step:4:260 which probably corresponds to https://github.com/DjangoGirls/tutorial/blob/e9ee3d53a452e35dc91d3f291050942f755dc7d4/.github/workflows/deploy.yml#L34

I wonder why that should fail, though, as https://github.com/DjangoGirls/tutorial/blob/e9ee3d53a452e35dc91d3f291050942f755dc7d4/.github/workflows/deploy.yml#L28 should install the package providing that command. https://github.com/DjangoGirls/tutorial/actions/runs/11323400245/job/31485981412#step:3:28 seems to indicate it was retrieved from a cache.

@ekohl Can you look into this?

**Comment by ekohl** [comment]
> rename

Confusingly enough, `rename` doesn't provide the `rename` binary according to https://packages.ubuntu.com/jammy/all/rename/filelist. There is also `rename.ul` from `util-linux` on Ubuntu 24.04 which should be present by default, but I noticed `ubuntu-latest` still points to 22.04 here.

My theory is that when the package wasn't cached it ran DPKG triggers, which sets up alternatives:
```console
Setting up rename (2.02-1) ...
update-alternatives: using /usr/bin/file-rename to provide /usr/bin/rename (rename) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/rename.1.gz because associated file /us
r/share/man/man1/file-rename.1p.gz (of link group rename) doesn't exist
```

Then the cache action may only cache the files but not run triggers.

I've submitted #1856 which I think is the easiest way to fix it now. Right after I submitted it I figured out why it probably broke, so I updated the message right after.
